docs/tools/offline/mbed-studio.md

```diff
@@ -0,0 +1,164 @@
+<h2 id="mbed-studio">Arm Mbed Studio</h2>
+
+Arm Mbed Studio is a local development environment for Mbed OS projects. This document helps Mbed OS developers use Mbed Studio to code, run, debug and deploy Mbed OS programs to Mbed Enabled devices. The Mbed Studio project is based on the Eclipse framework and extends Eclipse’s IDE capability to provide a more focused experience for developing with Mbed OS. In particular, it allows users to develop and deploy code offline. It also provides access to existing online resources.
+
+### Current status and downloads
+
+Mbed Studio is currently in closed alpha. You can request access here if you're happy to provide feedback. During this phase, some features are still in progress, and there are frequent updates to the product.
```

Comment:
@AnotherButler  I'd be extremely happy to provide feedback please.

Reply:
@JoeTheGuitarist Is this something you can help with?

Reply:
@loverdeg-ep we'll be putting up the signup form soon, so assuming you have an Mbed account, I'd be more than happy to give you access. Thanks!